ci/conda_env_cpp.yml

```diff
@@ -17,7 +17,7 @@
 
 aws-sdk-cpp
 benchmark=1.4.1
-boost-cpp>=1.68.0
```

Comment:
The lower-limit is no problem, I'm more confused why/where we have an upper limit. This will cause issues.